src/admin/admin.service.ts

```diff
@@ -22,8 +22,8 @@ export class AdminService {
     private readonly reportRepository: ReportRepository,
   ) {}
 
-  async getAllReports(): Promise<FullReportDto[]> {
-    const reports = await this.reportRepository.getAllReports();
+  async getAllReports(type: string): Promise<FullReportDto[]> {
```

Comment:
The term "type" is a reserved keyword in certain programming languages, such as Python and Scala. While it is generally recommended to steer clear of using reserved keywords as variable names to prevent potential conflicts.